src/test/java/tr/com/srdc/cda2fhir/AllergyConcernActTest.java

```diff
@@ -0,0 +1,391 @@
+	private static Map<String, String> cdaAllergyIntolaranceTypeCodeToName = new HashMap<String, String>();
+	
+	private static Map<String, Object> clinicalStatusMap = JsonUtils.filepathToMap("src/test/resources/jolt/value-maps/AllergyIntoleranceClinicalStatus.json");
+	private static Map<String, Object> verificationStatusMap = JsonUtils.filepathToMap("src/test/resources/jolt/value-maps/AllergyIntoleranceVerificationStatus.json");
+	private static Map<String, Object> categoryMap = JsonUtils.filepathToMap("src/test/resources/jolt/value-maps/AllergyIntoleranceCategory.json");
+		
+	@BeforeClass
+	public static void init() {
+		CDAUtil.loadPackages();
+		
+		cdaObjFactory = (ConsolFactoryImpl) ConsolFactoryImpl.init();
+		cdaTypeFactory = DatatypesFactoryImpl.init();		
+		cdaFactory = (CDAFactoryImpl) CDAFactoryImpl.init();
+		
+		cdaProblemStatusCodeToName.put("55561003", "Active");
+		cdaProblemStatusCodeToName.put("73425007", "Inactive");
+		cdaProblemStatusCodeToName.put("413322009", "Resolved");
+		
+		cdaAllergyIntolaranceTypeCodeToName.put("419199007", "Allergy to substance (disorder)");
+		cdaAllergyIntolaranceTypeCodeToName.put("416098002", "Drug allergy (disorder)");
+		cdaAllergyIntolaranceTypeCodeToName.put("59037007", "Drug intolerance (disorder)");		 
+		cdaAllergyIntolaranceTypeCodeToName.put("414285001", "Food allergy (disorder)");
+		cdaAllergyIntolaranceTypeCodeToName.put("235719002", "Food intolerance (disorder)");
+		cdaAllergyIntolaranceTypeCodeToName.put("420134006", "Propensity to adverse reactions (disorder)");
+		cdaAllergyIntolaranceTypeCodeToName.put("419511003", "Propensity to adverse reactions to drug (disorder)");		 
+		cdaAllergyIntolaranceTypeCodeToName.put("418471000", "Propensity to adverse reactions to food (disorder)");
+		cdaAllergyIntolaranceTypeCodeToName.put("418038007", "Propensity to adverse reactions to substance (disorder)");
+		cdaAllergyIntolaranceTypeCodeToName.put("232347008", "Dander (animal) allergy");		
+	}
+
+	static private AllergyIntolerance findOneResource(Bundle bundle) throws Exception {
+    	List<AllergyIntolerance> allergyResources = bundle.getEntry().stream()
+    			.map(r -> r.getResource())
+    			.filter(r -> (r instanceof AllergyIntolerance))
+    			.map(r -> (AllergyIntolerance) r)
+				.collect(Collectors.toList());
+    	Assert.assertEquals(1, allergyResources.size());
+    	return allergyResources.get(0);	
+	}
+	
+	static private void verifyAllergyIntoleranceCategory(AllergyProblemAct act, String expected) throws Exception {
+		Bundle bundle = rt.tAllergyProblemAct2AllergyIntolerance(act);
+		AllergyIntolerance allergyIntolerance = findOneResource(bundle);
+
+    	Enumeration<AllergyIntolerance.AllergyIntoleranceCategory> category = allergyIntolerance.getCategory().get(0);
+    	String actual = category == null ? null : category.asStringValue();
+		Assert.assertEquals(expected, actual);		
```

Comment:
do you mind putting in textual descriptions of the asserts similar to how I did in my PR? Helps to just define the assertions themselves.

Reply:
ok

Reply:
i put messages